samples/usb/Makefile

```diff
@@ -0,0 +1,5 @@
+XBE_TITLE = nxdk\ sample\ -\ usb
```

Comment:
nit: I think just calling this sample "usb" is not ideal; but I have trouble coming up with my own name.

*(I'd assume there'll be different USB samples eventually, at least some of which will likely deserve a usb prefix)*

Reply:
Perhaps call it `hotplugging`?

Reply:
Agree. This sample really just shows the absolute basic core init so it atleast builds during ci tests.
I am working on an 'all-in-one` usb sample. https://github.com/Ryzee119/nxdk_ohci_test/blob/xbox/main.c, advantage being it includes all source files in ci runs to catch issues better.